app/react-native/src/preview/index.js

```diff
@@ -1,24 +1,25 @@
 /* eslint-disable no-underscore-dangle */
 
 import React from 'react';
-import { AsyncStorage, NativeModules } from 'react-native';
+import { AsyncStorage, NativeModules, Platform } from 'react-native';
 import parse from 'url-parse';
 import addons from '@storybook/addons';
 
 import Events from '@storybook/core-events';
 import Channel from '@storybook/channels';
 import createChannel from '@storybook/channel-websocket';
-import { StoryStore, ClientApi } from '@storybook/core/client';
-import OnDeviceUI from './components/OnDeviceUI';
-import StoryView from './components/StoryView';
+import { StoryStore, ClientApi } from '@storybook/client-api';
+import OnDeviceUI from './components/OnDeviceUI/index';
```

Comment:
index implicit

Reply:
Right, but makes it harder to find, if neglected.